libs/blocks/marketo/marketo.js

```diff
@@ -23,13 +23,20 @@ const MUNCHKIN_ID = 'munchkin id';
 
 /* Marketo adds default styles that we want to remove */
 const cleanStyleSheets = (baseURL) => {
-  const { styleSheets } = document;
+  const { styleSheets, head } = document;
 
   [...styleSheets].forEach((sheet) => {
     if (sheet.href?.includes(baseURL)) {
       sheet.disabled = true;
     }
   });
+
+  // Removing Custom CSS from Marketo.
+  head.querySelectorAll('style').forEach((s) => {
```

Comment:
What styles are you seeing here?

Reply:
<img width="1455" alt="image" src="https://user-images.githubusercontent.com/55804872/199334916-938ea0b0-9fb1-45fc-815d-413ef2ca534e.png">